packages/frontend-2/components/settings/workspaces/security/RemoveDialog.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <LayoutDialog
+    v-model:open="isOpen"
+    title="Remove domain"
+    max-width="sm"
+    :buttons="dialogButtons"
+  >
+    Are you sure you want to remove
+    <b>@{{ domain.domain }}</b>
```

Comment:
We generally use `<span class="font-semibold"` for bold texts. You can wrap all text in `<p class="text-body-xs text-foreground">` to make sure it's the right color/size

Reply:
this did feel quite retro haha thank you for the details on how we do this in 2024

Reply:
![949627e7-e639-4a74-828c-1cab81027bd4_text](https://github.com/user-attachments/assets/83aaeff9-65b2-40b3-b2f6-ce668410d883)